src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -204,6 +204,12 @@ public static bool IsAnyToken(SyntaxKind kind)
                 case SyntaxKind.InterpolatedVerbatimStringStartToken:
                 case SyntaxKind.InterpolatedStringTextToken:
                 case SyntaxKind.InterpolatedStringEndToken:
+                case SyntaxKind.RawInterpolationOpenToken:
+                case SyntaxKind.RawInterpolationCloseToken:
+                case SyntaxKind.MultiLineRawInterpolatedStringStartToken:
+                case SyntaxKind.MultiLineRawInterpolatedStringEndToken:
+                case SyntaxKind.SingleLineRawInterpolatedStringStartToken:
+                case SyntaxKind.SingleLineRawInterpolatedStringEndToken:
```

Comment:
FYI, I added a note to test plan to review usages of `SyntaxKind.InterpolatedStringStartToken` in IDE/Workspaces layer (there's a few)

Reply:
yup yup!

Reply:
tagging @allisonchou  as well.